includes/class.llms.frontend.assets.php

```diff
@@ -205,6 +204,10 @@ public static function enqueue_scripts() {
 			'llms-ajaxurl',
 			'window.llms = window.llms || {};window.llms.ajaxurl = "' . admin_url( 'admin-ajax.php', $ssl ) . '";'
 		);
+		self::enqueue_inline_script(
```

Comment:
I was thinking about this too... All the inline stuff could be it's own class. I was thinking about making an `LLMS_Assets_Inline` class in favor of adding the inline stuff to the assets class. Actually, maybe it would make more sense to make inline asset-related functionality a trait that gets mixed into the asset class. I'm thinking mostly of keeping the number of methods per file smaller.

What do you think?

Reply:
I'm going to move this conversation into #1292 so I can merge this into that PR

Reply:
You know I'm not sure what this LLMS_Assets_Inline would do.
Not sure if semantically it makes sense making a class for that, I understand keeping the number of methods per file smaller though.
Traits can be good, but you know we'll only use them here. 
Will the Inline stuff add so many methods?